test/parallel/test-http.js

```diff
@@ -121,12 +121,12 @@ server.on('listening', function() {
                              ['Cookie: abc=123',
                               'Cookie: def=456',
                               'Cookie: ghi=789']);
-      assert.strictEqual(res.statusCode, 200);
+      assert.strictEqual(200, res.statusCode);
       let body = '';
       res.setEncoding('utf8');
       res.on('data', (chunk) => { body += chunk; });
       res.on('end', common.mustCall(() => {
-        assert.strictEqual(body, 'The path was /world');
+        assert.strictEqual('The path was /world', body);
```

Comment:
Hm … this changes a lot of cases that were previously the “right” way around – the expected value (usually the constant) should be the second argument?

Reply:
Ah okay!  Thanks for the heads up!